app/src/main/java/com/codesoom/assignment/CrudHttpHandler.java

```diff
@@ -0,0 +1,116 @@
+package com.codesoom.assignment;
+
+import com.codesoom.HttpEnum.HttpMethodCode;
+import com.codesoom.models.SendResponseData;
+import com.codesoom.models.TaskService;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import java.net.URI;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CrudHttpHandler implements HttpHandler {
+
+    private static final TaskService TASK_SERVICE_IMPL = new TaskService();
```

Comment:
대문자와 `_`로 이름을 지어주는 건 보통 숫자나 문자열 상수값입니다. 여기에서는 이렇게 이름을 지어주세요.
```suggestion
    private static final TaskService taskServiceImpl = new TaskService();
```

Reply:
네 알겠습니다 !